pdu/Cube-F7-Src-respin/Src/tim.c

```diff
@@ -99,6 +100,39 @@ void MX_TIM6_Init(void)
 
   /* USER CODE END TIM6_Init 2 */
 
+}
+/* TIM9 init function */
+void MX_TIM9_Init(void)
+{
+
+  /* USER CODE BEGIN TIM9_Init 0 */
+
+  /* USER CODE END TIM9_Init 0 */
+
+  TIM_ClockConfigTypeDef sClockSourceConfig = {0};
+
+  /* USER CODE BEGIN TIM9_Init 1 */
+
+  /* USER CODE END TIM9_Init 1 */
+  htim9.Instance = TIM9;
+  htim9.Init.Prescaler = 0;

```

Comment:
I don't think this is what you intended. TIM9 runs at 200 MHz. With a prescaler of 0, the timer will be 1 tick/5 ns. Go to CubeMX and adjust the prescaler to 200-1 to get 1 MHz (1 us).